udata_front/theme/gouvfr/templates/macros/metadata.html

```diff
@@ -20,7 +20,8 @@
 <meta name="static-root" content="{{ static('', _burst=False) }}" />
 <meta name="api-root" content="{{ url_for('api.root') }}" />
 <meta name="api-2-root" content="{{ url_for('apiv2.root') }}" />
-<meta name="admin-root" content="{{ url_for('admin.index') }}" />
+<meta name="admin-root" content="{{ beta_admin_url_for(path='/', fallback_path='') }}" />
```

Comment:
Do we want to change this one too ? My understanding was that this one would still point to the old admin for the other reusers only.

We can't have the same links to the old and new admins because the old one uses singular and the new one uses plural.

Reply:
It will point to the old admin if there is no `NEW_ADMIN_URL`